dev-packages/e2e-tests/test-applications/nextjs-app-dir/tests/middleware.test.ts

```diff
@@ -76,6 +76,8 @@ test('Should trace outgoing fetch requests inside middleware and create breadcru
           'http.response.status_code': 200,
           type: 'fetch',
           url: 'http://localhost:3030/',
+          'http.url': 'http://localhost:3030/',
+          'server.address': 'localhost:3030',
```

Comment:
@gggritso regarding port, you can see here that the port is part of the server.address if it is in the URL (=non-standard)

Reply:
Roger, thanks!